src/components/Forecast/Widget.jsx

```diff
@@ -0,0 +1,181 @@
+import React from 'react'
+import Form from './Form/Form'
+import WeatherDetailsToday from './WeatherDetailsToday/WeatherDetailsToday'
+import FiveDayForecast from './FiveDayForecast/FiveDayForecast'
+
+import './Widget.scss'
+
+const myAPIKey = 'dce6d076b7a2ffe64e08efe5ff779fa3'
+
+class Widget extends React.Component {
+  constructor(props) {
```

Comment:
In this example you actually don't need this constructor at all! Look at the two comments below to see how and why 🙂